lib/common/strings.c

```diff
@@ -894,27 +884,17 @@ pcmk__parse_ll_range(const char *srcstring, long long *start, long long *end)
 gboolean
 pcmk__str_in_list(GList *lst, const gchar *s, uint32_t flags)
 {
-    GCompareFunc fn;
-
     if (lst == NULL) {
         return FALSE;
     }
```

Comment:
Technically we don't even need this test anymore :)

Reply:
Ah you're right.  Fixed.  There's not much left of that function.